### DISCUSSION TITLE
mybinder pasting .jl into new notebook give errors

### DISCUSSION THREAD

**Discussion by paradocs** [JuliaPluto/PlutoTest.jl#30] [Issues] [answered]
We start a Pluto mybinder and "Create a new notebook"
We paste in a previously saved notebook and run it.
1) We get an error in cell 1 because 
	using Markdown
	using InteractiveUtils
do not have       begin           end
Solution: Add begin end  around default using statements.

2) Again with pasting a previously saved notebook:
we must delete the last two cells named
PLUTO_PROJECT_TOML_CONTENTS
PLUTO_MANIFEST_TOML_CONTENTS
I believe that these entries are recreated.
We can have our students delete these cells before
saving but it would be nice if this was automated.

Thank you for making Pluto notebooks a great
teaching tool.

**Comment by fonsp** [comment, marked as answer]
Hey David!

Instead of **first** clicking _New notebook_, and **then** pasting the notebook contents, Pluto wants you to paste the notebook contents **directly into the main menu**, so without clicking *New notebook*. 

Hope this helps!

-fons
